### PR TITLE
Use kotlin.time API

### PR DESCRIPTION
**Pull request by pubiqq**
Kotlin Time API is now [stable](https://kotlinlang.org/docs/whatsnew19.html#stable-time-api), so it's time to replace the self-made time API with it.

`KoinPlatformTimeTools` and other parts of "koin time API" have been marked as deprecated and hopefully they will be removed in the future, as they shouldn't have been part of the public API.

### CODE REVIEW

**Comment by pubiqq** [comment]
Friendly bump

**Comment by pubiqq** [comment]
Friendly bump

**Comment by pubiqq** [comment]
Friendly ping

**Comment by pubiqq** [comment]
Friendly ping

**Comment by pubiqq** [comment]
Friendly ping

File: examples/jvm-perfs/src/main/kotlin/org/koin/benchmark/PerfRunner.kt [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
-        return PerfResult(avgStart,avgExec)
+        println("Avg start: $avgStartInMs ms")
+        println("Avg execution: $avgExecInMs ms")
+        return PerfResult(avgStartInMs, avgExecInMs)
     }
 
-    @OptIn(KoinExperimentalAPI::class)
-    fun runScenario(index: Int): Pair<Double, Double> {
-        val (app, duration) = measureDurationForResult {
+//    @OptIn(KoinExperimentalAPI::class)
```
Comment (pedrofsn) [inline examples/jvm-perfs/src/main/kotlin/org/koin/benchmark/PerfRunner.kt:24]:
```suggestion
```

**Review by pedrofsn — APPROVED** [review]
I just looked the code in github code review tool. To me it's okay. But I think @arnaudgiuliani need to double check it. ;)

**Comment by pubiqq** [comment]
Bump

**Comment by pubiqq** [comment]
Bump

**Pull request closed without merging**